### DISCUSSION TITLE
Wallet Core for IONIC Framework?

### DISCUSSION THREAD

**Discussion by TonyCicero** [trustwallet/wallet-core#1385] [Q&A] [answered]
Hi there,
We are implementing a feature into our Ionic project that will require users to send and receive ERC-20 Tokens. Is there an Sdk available that will work with Ionic? Also, is there a contact email to get in touch with Trust Wallet?

**Comment by hewigovens** [comment, marked as answer]
we don't support ionic officially, but you can wrap it into a plugin like this: https://github.com/johnnynanjiang/TrustWalletCoreCordovaPluginSwift

**Reply by TonyCicero** [reply]
Great thanks!  Do you have any resources you could recommend for Android as well?
